packages/permission-controller/src/SubjectMetadataController.ts

```diff
@@ -235,11 +232,9 @@ export class SubjectMetadataController extends BaseController<
    */
   trimMetadataState(): void {
     this.update((draftState) => {
+      // @ts-expect-error ts(2589)
       return SubjectMetadataController.getTrimmedState(
-        // Typecast: ts(2589)
-        // TODO: Replace `any` with type
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        draftState as any,
+        draftState,
```

Comment:
Same question as above, can this not be `unknown`?

Reply:
I don't understand the question. At present this type is `WritableDraft<SubjectMetadataControllerState>`, which is exactly what we want.

Reply:
middle of the night mobile reviews lol, apologies I misread!